### PR TITLE
Alert component

### PR DESCRIPTION
**Pull request by KoushikKintali**
Alert Component added

https://nan-ui.netlify.app/components/alert/alert.html

### CODE REVIEW

**Review by bipin7yadav — APPROVED** [review]
Looks good to me

File: components/alert/alert.html [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+                            <span class="material-icons-outlined">check_circle</span>
+                            A simple success alert
+                            <span class="material-icons-outlined btn-close">close</span>
+                        </div>
+                    </div>
+                </div>
+                <div class="code-block">
+                    <iframe
+                        src="https://carbon.now.sh/embed/i5ALuWw0j8m8II6V676y"
+                        style="width: 100%; height: 287px; border:0; transform: scale(1); overflow:hidden;"
```
Comment (gopalbbharadva) [inline components/alert/alert.html:109]: Suggestion: Remove inline styles and keep these properties into class.

**Review by gopalbbharadva — APPROVED** [review]

File: components/alert/alert.css [hunk 2]
```diff
@@ -0,0 +1,69 @@
+:root {
```
Comment (mimansha-swarup) [inline components/alert/alert.css:1]: you can keep it in a separate file and link that file

**Review by mimansha-swarup — COMMENTED** [review]
except that all looks good

**Pull request merged**